src/main/java/com/thecookiezen/presentation/ClusterController.java

```diff
@@ -43,20 +44,41 @@ public String page() {
         return "clusters/all";
     }
 
-    @RequestMapping(value = "clusters", method = RequestMethod.POST)
-    public String saveOrUpdate(@Valid Cluster cluster, BindingResult bindingResult) {
+    @RequestMapping(value = "clusters", method = RequestMethod.DELETE)
+    public String delete(@RequestParam Long clusterId) {
+        clusterRepository.remove(clusterId);
+        return "redirect:/clusters/all";
+    }
+
+    @RequestMapping(value = "clusters/newhost", method = RequestMethod.GET)
+    public String newHostForm(@RequestParam Long clusterId, Model model) {
+        model.addAttribute("clusterId", clusterId);
+        model.addAttribute("host", new DockerHost());
```

Comment:
\[Checkstyle\] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4\.